public/portfolio_shared_data.json

```diff
@@ -5,13 +5,13 @@
       "social": [
         {
           "name": "github",
-          "url": "https://github.com",
+          "url": "https://github.com/DavidMatalik",
```

Comment:
that looks good, I have a comment about it in the other PR that's now disregarded! :))